app/javascript/article-form/components/EditorActions.jsx

```diff
@@ -74,7 +74,8 @@ EditorActions.propTypes = {
   edited: PropTypes.bool.isRequired,
   version: PropTypes.string.isRequired,
   onClearChanges: PropTypes.func.isRequired,
-  passedData: PropTypes.string.isRequired,
+  passedData: PropTypes.oneOfType([PropTypes.object, PropTypes.string])
+    .isRequired,
```

Comment:
is there a scenario where `passedData` is actually a string? it's been passed `this.state` which AFAIK it's always an object, so we probably just need to switch the type here

Reply:
Okay I'll do that